jsonschema2pojo-core/src/main/java/org/jsonschema2pojo/rules/ObjectRule.java

```diff
@@ -64,6 +65,9 @@
  */
 public class ObjectRule implements Rule<JPackage, JType> {
 
+    /** The toplevel classes per schema URI. Used when generating inner classes. */
+    private static final Map<URI, JDefinedClass> topLevelClasses= new HashMap<>();
```

Comment:
I am not quite happy with having to use a static map here. But since there are several ObjectRule objects created in the generation process an instance field isn't suitable here.